src/lib/crypto/kimchi_backend/common/plonk_constraint_system.ml

```diff
@@ -888,6 +895,17 @@ end = struct
   open Core_kernel
   open Pickles_types
 
+  (* Use by compute_witness to build the runtime tables from the Lookup
```

Comment:
Typo fixed in https://github.com/MinaProtocol/mina/pull/14048. I preferred to merge this one to be able to build on top of it instead of waiting again for the CI :).